src/app/external-cluster-wizard/steps/external-cluster/provider/aks/component.ts

```diff
@@ -73,6 +73,11 @@ enum Mode {
   User = 'User',
 }
 
+export enum LocationState {
```

Comment:
Should we have a 3rd state for no locations? Same comment for other states being added in this PR. This can be enhanced in future so if you think it will more time please create an issue.

Reply:
I think we didn't need 3rd state since Placeholder for Ready (when list) exists or Empty we need to show same but yes we can can create if need to show 3rd state WDYT?

Reply:
In regular cluster wizard we have 3 states for all dropdowns.

Reply:
Than i will be creating a sep ticket to manage in all other places too. Thanks